### ISSUE TITLE
Gold text failing contrast accessibility test

### ISSUE THREAD

**Issue by saracope** [GSA/opioids.gov#53]
The gold color using for captions in the Crisis and Illicit Supply sections is failing the accessibility contrast check. 

Possible fix:
1.) Bump up caption size
2.) Darken the gold color
3.) Put caption on darker background instead of white
4.) Combination of those above options

Here's the [WCAG2.0 AA](https://www.w3.org/TR/WCAG20/) guideline:
1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1.
